packages/core/content-releases/admin/src/pages/tests/ReleasesSettingsPage.test.tsx

```diff
@@ -0,0 +1,93 @@
+import { useRBAC } from '@strapi/admin/strapi-admin';
+import { render, server, screen } from '@tests/utils';
+import { rest } from 'msw';
+
+import { ReleasesSettingsPage, ProtectedReleasesSettingsPage } from '../ReleasesSettingsPage';
+
+jest.mock('@strapi/admin/strapi-admin', () => ({
+  ...jest.requireActual('@strapi/admin/strapi-admin'),
+  useRBAC: jest.fn(() => ({
+    isLoading: false,
+    allowedActions: { canRead: false, canUpdate: false },
+  })),
+}));
+
+describe('Releases Settings page', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders the no permissions content if you do not have read permission', async () => {
+    // @ts-expect-error – mocking
+    useRBAC.mockImplementation(() => ({
+      isLoading: false,
+      allowedActions: { canRead: false, canUpdate: false },
+    }));
+
+    render(<ProtectedReleasesSettingsPage />);
+
+    expect(
+      screen.getByText("You don't have the permissions to access that content")
+    ).toBeInTheDocument();
+  });
+
+  it('renders the settings page with read permission', async () => {
+    // @ts-expect-error – mocking
+    useRBAC.mockImplementation(() => ({
+      isLoading: false,
+      allowedActions: { canRead: true, canUpdate: false },
+    }));
+    server.use(
+      rest.get('/content-releases/settings', (req, res, ctx) =>
+        res(
+          ctx.json({
+            data: {
+              defaultTimezone: null,
+            },
+          })
+        )
+      )
+    );
+
+    render(<ReleasesSettingsPage />);
```

Comment:
I don't think it's required to mock RBAC if you're rendering the non-protected component. I would recommend to always render the protected version since you already have the mock set up. You wouldn't even need to export ReleasesSettingsPage directly